erpnext_documentation/www/docs/user/manual/en/manufacturing/manufacturing-dashboard.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+#Manufacturing Dashboard
+
+To access the Manufacturing Dashboard, go to:
+
+> Home > Customization > Dashboard > Manufacturing
+
+<img class="screenshot" alt="Manufacturing Dashboard" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard.png">
+
+## Number Cards
+
+<img class="screenshot" alt="Number Cards" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-number-cards.png">
+
+1. Monthly Total Work Order :- The user will get the total count of Work Orders which are in Open, In Progress, and Completed state. The creation of the Work Orders is one month back from the current date will be displayed in the number card.
+1. Monthly Completed Work Order :- The user will get the total count of Work Orders which are in the Completed state. The creation of the Work Orders is one month back from the current date will be displayed in the number card.
+1. Ongoing Job Card :- The user will get the total count of Job Cards that are not in the Completed state.
+1. Monthly Quality Inspection :- The user will get the total count of Quality Inspection records which are in the Submitted state. The creation of the work orders is one month back from the current date will be displayed in the number card.
+
+## Dashboard
+
+<img class="screenshot" alt="Produced Quantity & Completed Operation" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-2.png">
+
+### Produced Quantity
+
+The chart will give the information about the total number of quantity produced (using work order) in the last year based on Quarterly. Users can also view the chart data based on Daily, Weekly, Monthly, Quarterly, Yearly basis.
+
+### Completed Operation
+
+The chart will give information about the total number of operations completed in the last year based on Quarterly. Users can also view the chart data based on Daily, Weekly, Monthly, Quarterly, Yearly basis.
+
+<hr>
+
+<img class="screenshot" alt="Work Order & Quality Inspction Analysis" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-3.png">
+
+### Work Order Analysis
+
+The chart will give the information about the number of Work Orders based on Not Started, In Process, Stopped, Completed status. The chart type is a Donut and it will provide the information based on the last year's work order data. Users can also change the date range by clicking on the Filter button.
+
+### Quality Inspction Analysis
+
+The chart will give information about the number of Quality Inspections based on Accepted and Rejected status. The chart type is a donut and it will provide the information based on the last year's quality inspection data. Users can change the date range by clicking on the Filter button.
```

Comment:
```suggestion
This chart will give information about the number of Quality Inspections based on Accepted and Rejected status. The chart type is a donut and it will provide the information based on the last year's quality inspection data. Your can change the date range by clicking on the **Filter** button.
```